mautic/mautic
# Support label_html value.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Support the use of label_html values. This would allows us to replace labels where appropriate by using HTML for them. 
This could enable to add links to labels, to for example point users to add segments, on places they should add something to a segment. 

The use of label_html is already a defined twig form_attribute. https://symfony.com/doc/current/reference/forms/types/form.html#label-html

Other example: For instance, when I'm importing a list of contacts, I always forget to create a segment first. I'm always starting the import config, going back to segments, create a segment and back to import. If we could use above, we could add a link to segment creation in the import to segment label field. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Alter a form element. Add label_html => true and replace the label to something with HTML in it. `<h1>TEST</h1>`. See that when adding the label_html value (and set to true) the html in the label value is not escaped.
